Options.lua

```diff
@@ -8,7 +8,7 @@ The commandline options understood by the framework:
 	-a <path>      -- Path to the AutoAPI description files (by Cuberite's BindingsProcessor)
 	-c             -- Check by clearing each API object after each callback
 	-e <filename>  -- Load file as extra API description (ManualAPI.lua by ManualApiDump plugin)
-	-f             -- Fuzz command handlers (send various garbate to them to test their user-input resilience)
+	-f <pattern>   -- Prefills the sandbox (_G) if the name of the variable matches the provided pattern(s).
```

Comment:
This is somewhat unclear. How about `Prefills the sandbox with API symbols matching the specified pattern, instead of creating them as-needed (#20).`
I think it'd be good to include the PR number for reference.